.github/workflows/build.yml

```
@@ -101,6 +101,7 @@ jobs:
       fail-fast: false
       matrix:
         compiler: ['gcc-11 g++-11', 'clang clang++']
+        build_option: ['BUILD_64BIT=OFF', 'BUILD_64BIT=ON']
```

[IMO]
`build_option`はちょっと曖昧すぎる気がします
```suggestion
        build_bit: ['BUILD_64BIT=OFF', 'BUILD_64BIT=ON']
```